projects/plugins/jetpack/json-endpoints/class.wpcom-json-api-comment-endpoint.php

```diff
@@ -71,7 +71,17 @@ public function get_comment( $comment_id, $context ) {
 			return new WP_Error( 'unknown_comment', 'Unknown comment', 404 );
 		}
 
-		$types = array( '', 'comment', 'pingback', 'trackback', 'review' );
+		/**
+		 * Filter the comment types that are allowed to be returned.
+		 *
+		 * @since $$next-version$$
+		 *
+		 * @module json-api
+		 *
+		 * @param array $types Array of comment types.
+		 */
+		$types = apply_filters( 'wpcom_json_api_comment_types', array( '', 'comment', 'pingback', 'trackback', 'review' ) );
```

Comment:
Since that's a new endpoint, let's use the `jetpack_` prefix. While changing existing filter names is hard, we can switch to a consistent prefix for new filters, which is recommended by the Plugins team.

```suggestion
		$types = apply_filters( 'jetpack_json_api_comment_types', array( '', 'comment', 'pingback', 'trackback', 'review' ) );
```